packages/strapi-global-object/strapi-global.ts

```diff
@@ -0,0 +1,20 @@
+import knex from "knex";
+
+const strapiObject = {
+  plugins: {},
+  components: {},
+  contentTypes: {},
+  query: (s: any) => s,
+  models: {},
+  connections: {
+    default: knex,
+  },
+  db: {},
+  app: {
+    use: (x: any) => x,
+  },
+};
+
+module.exports = {
```

Comment:
this module should be customisable, consider to use class with customisation done in constructor?